project/docs/conf.py

```diff
@@ -0,0 +1,246 @@
+# -*- coding: utf-8 -*-
+#
+# IoC documentation build configuration file, created by
+# sphinx-quickstart on Fri Mar 29 01:43:00 2013.
+#
+# This file is execfile()d with the current directory set to its containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+import sys, os
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#sys.path.insert(0, os.path.abspath('.'))
+
+# -- General configuration -----------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+#needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be extensions
+# coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
+extensions = ['sensio.sphinx.refinclude', 'sensio.sphinx.configurationblock', 'sensio.sphinx.phpcode']
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# The suffix of source filenames.
+source_suffix = '.rst'
+
+# The encoding of source files.
+#source_encoding = 'utf-8-sig'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'Sonata ~ AdminBundle'
+copyright = u'2010-2015, Thomas Rabaix'
```

Comment:
we should change this to `2010-2018`

Reply:
Or just 2010 IIRC

Reply:
https://github.com/sonata-project/dev-kit/blob/master/project/LICENSE#L3 yep, just 2010, should I fix the style thing in a different PR?

Reply:
No we can fix it here IMO